### ISSUE TITLE
Correct training of YOLOv5-P6 models with custom data set

### ISSUE THREAD

**Issue by haimat** [ultralytics/yolov5#3081]
I want to train a YOLOv5-P6 model on a custom data set.
So after preparing the images and labels I start training like this (using 4 GPUs in parallel):

    python3 -m torch.distributed.launch --nproc_per_node 4 train.py --img 1280 --batch 16 --epochs 50 --data ../img/yolo.yaml --weights yolov5x6.pt

After the training script has started and printed the model summary, I get the following output:

````
Model Summary: 773 layers, 141090828 parameters, 141090828 gradients, 221.5 GFLOPS
Transferred 1004/1012 items from yolov5x6.pt
Scaled weight_decay = 0.0005
Optimizer groups: 171 .bias, 171 conv.weight, 167 other
````

Now I am wondering, is this correct that only 1004/1012 items have been transferred from `yolov5x6.pt`?

**Comment by glenn-jocher** [comment]
@haimat everything is correct. All layers with matching names and shapes are transferred. Since your custom dataset likely does not contain 80 classes it is not possible to transfer output layers.

**Comment by haimat** [comment]
@glenn-jocher Makes much sense, thanks a lot!
